### PR TITLE
proj: OpenML

### PR DESCRIPTION
**Pull request by OscarAblinger**
OpenML is kinda interesting for our list. There is no direct benefit for any of the goals, so I'm not sure whether it should be added.
The argument for adding it, would be that it tries to have an impact on society by being a ressource for everyone. However, I'm not sure how it differenciates itself from some of the other options and how much that really furthers society in a greater way.

The arguments against adding it, is that – while useful – it doesn't really improve the lives of anyone. Additionally, it also does not fit into any SDG (I added it to SDG 4: Quality education, but if you look at it's [targets](https://sustainabledevelopment.un.org/sdg4#targets), it fits to none of them)

So, in short, I'd advice against adding it, but thought, I'd still create this PR in case that you disagree.

### CODE REVIEW

**Comment by AdrienLemaire** [comment]
Indeed, it's difficult to find an argument for accepting this project ^^'
